src/herder/TransactionQueue.cpp

```diff
@@ -196,6 +202,14 @@ TransactionQueue::canAdd(TransactionFrameBasePtr tx,
                     return TransactionQueue::AddResult::ADD_STATUS_DUPLICATE;
                 }
 
+                // By this point, there's already a tx in the queue for this
+                // account, and only the tx with the lowest seqnum for an
+                // account can have non-zero values for these fields.
+                if (tx->getMinSeqAge() != 0 || tx->getMinSeqLedgerGap() != 0)
+                {
+                    return TransactionQueue::AddResult::ADD_STATUS_ERROR;
```

Comment:
`ADD_STATUS_ERROR` can only be used in conjunction with tx error codes like `tx->getResult().result.code(txINSUFFICIENT_BALANCE)`.
I think `ADD_STATUS_TRY_AGAIN_LATER` is probably a better one here

Reply:
actually the appropriate error code is `txTOO_EARLY` as this is what we'd return if the previous transaction was applied (and is consistent with `checkValid`)

Reply:
This check also needs to be done for a fee bump with a new inner tx.